src/main.py

```diff
@@ -11,19 +13,21 @@ def setup_logging():
     if not os.path.exists("logs"):
         os.makedirs("logs")
 
-    # Get current datetime formatted as a string
+    logger = logging.getLogger(__name__)
+
+    # Configure logging
+    with open("logging.json") as logging_conf_file:
+        logging_conf = json.load(logging_conf_file)
+
+    # Dynamically set log file name to current time
     current_datetime = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+    logfile = logging_conf['handlers']['file']['filename']
+    logging_conf['handlers']['file']['filename'] = logfile.replace("{current_datetime}", current_datetime)
```

Comment:
Could you change the quotes on 24 and 25 to double quotes just for consistency?

Reply:
https://github.com/astral-sh/ruff 👀

Reply:
Agreed, fixed in ed271c4